runtime/vm/UpcallThunkMem.cpp

```diff
@@ -155,20 +191,19 @@ allocateThunkHeap(J9UpcallMetaData *data)
 
 	Trc_VM_allocateThunkHeap_Entry(thunkSize);
 
+	/* Create a thunk heap node of the list */
```

Comment:
Please finish each sentence with a period.

Reply:
Updated everywhere in this PR.